app/views/planning_application/local_policies/_comment.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if @local_policy.review_local_policies.where.not(reviewer_comment: "").any? %>
+  <div class="govuk-inset-text" id="reviewer_comment">
+    <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0">
+      Reviewer comment:
+    </p>
+    <% @comment = @local_policy.review_local_policies.where.not(reviewer_comment: nil).order(:created_at).last %>
```

Comment:
Similar issue here - we've got `nil` instead of `""` so unclear whether it's going to hit the same set of records. Again might be quicker to do in memory if the comments are already loaded.